### PR TITLE
Bug 1618578 - All perfherder dropdowns have the items in alphabetical order

### PR DESCRIPTION
**Pull request by kimberlythegeek**
Removed alphabetical sorting from dropdowns

### CODE REVIEW

File: ui/perfherder/graphs/TestDataModal.jsx [hunk 1]
```diff
@@ -399,8 +399,8 @@ export default class TestDataModal extends React.Component {
       {
         options: projectOptions,
```
Comment (ionutgoldan) [inline ui/perfherder/graphs/TestDataModal.jsx]: As I said, we need to alphabetically sort the rest of the platforms.

**Comment by armenzg** [comment]
Somehow code owners flagged us for review. Removing and unsubscribing from the PR.

**Comment by ionutgoldan** [comment]
@kimberlythegeek could you rebase this on latest master & resolve the conflicts?

File: ui/shared/DropdownMenuItems.jsx [hunk 2]
```diff
@@ -31,19 +31,13 @@ const DropdownMenuItems = ({
   <DropdownMenu className="overflow-auto dropdown-menu-height">
     {/* Items pinned to top of dropdown */}
     {pinned.length > 0 &&
```
Comment (ionutgoldan) [inline ui/shared/DropdownMenuItems.jsx:33]: We wrote the `pinned.length` condition 2 times.

Could you make only one condition out of it & then return the `pinned.map()` and the `<DropdownItem divider />`?

File: ui/shared/DropdownMenuItems.jsx [hunk 3]
```diff
@@ -30,20 +30,17 @@ const DropdownMenuItems = ({
 }) => (
   <DropdownMenu className="overflow-auto dropdown-menu-height">
     {/* Items pinned to top of dropdown */}
-    {pinned.length > 0 &&
-      options
-        .filter((item) => pinned.includes(item))
-        .sort((a, b) => a > b)
-        .map((item) =>
+    {pinned.length > 0 && (
+      <span>
```
Comment (ionutgoldan) [inline ui/shared/DropdownMenuItems.jsx:34]: I think a `React.Fragment` would suite this better than a `span`.

Comment (kimberlythegeek) [reply ui/shared/DropdownMenuItems.jsx:34]: ah, okay, I've never used Fragment before (though I've seen it in the code)

I just knew it had to have one containing element, I'll make that change

**Pull request merged**